virtcontainers/utils/utils_linux.go

```diff
@@ -71,6 +72,12 @@ func FindContextID() (*os.File, uint64, error) {
 		return nil, 0, err
 	}
 
+	if runtime.GOARCH == "ppc64le" {
```

Comment:
@alicefr  @justin-he @jongwu (/cc @Pennyzct ) - I guess we should check one of these (the default??) is correct for the other arch's as well..

Reply:
@grahamwhaley: I guess its the same for s390x and x86. This [code](https://github.com/kata-containers/runtime/issues/1943#issuecomment-521117461) suggested its different for power.  cc @alicefr @jodh-intel

Reply:
@nitkon - I think what @jodh-intel is asking for (and is the definitive 'source'), would be for you to show us exactly where in the Linux kernel source tree the difference is between power and all the other architectures.... somewhere maybe in https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/powerpc and show the x86 equivalent .... I think we need to see the root source of where the difference is coming from (just for everbodies info, and so they can then check the other arch's).

Reply:
@grahamwhaley I'm not familiar with it, spare me some time to check it on arm64.

Reply:
where is the prefix "8" instead of the "4" coming from?

Reply:
> @alicefr @justin-he @jongwu (/cc @Pennyzct ) - I guess we should check one of these (the default??) is correct for the other arch's as well..

IMO, the macro definition _IOW on arm64 is arch generic. i.e. VHOST_VSOCK_SET_GUEST_CID=4008af60, 
I search the kernel codes, seems only Powerpc has different definition of _IOC_WRITE.
At least, for arm arch, there is no need to add a specific code path

Reply:
I think it's the same for s390x. No need

Reply:
@nitkon have ever vsocket worked on power?

Reply:
@alicefr : Yes it did. Please check this [comment](https://github.com/kata-containers/runtime/issues/1943#issuecomment-523350547)

Reply:
mmh, did they change the `_IOC_WRITE ` for power in the mean time or how did it worked before?

Reply:
@alicefr : Kata with vsock was never tested before on Power. This is the first time we are enabling it.